app/plugin.go

```diff
@@ -655,6 +646,23 @@ func (a *App) mergeLocalPlugins(remoteMarketplacePlugins map[string]*model.Marke
 	return nil
 }
 
+func (a *App) getBaseMarketplaceFilter() *model.MarketplacePluginFilter {
+	filter := &model.MarketplacePluginFilter{
+		ServerVersion: model.CurrentVersion,
```

Comment:
Not sure if this is cleaner or what you have below, but just a suggestion 🤷‍♂️.
```suggestion
		ServerVersion: model.CurrentVersion,
		BuildEnterpriseReady: model.BuildEnterpriseReady == "true",
```

Reply:
I'm personally like the way it is right now. But that is very subjective. Objections on leaving it as it is?